src/Spice86.Core/Emulator/InterruptHandlers/Input/Keyboard/KeyboardInt16Handler.cs

```diff
@@ -43,7 +43,21 @@ public KeyboardInt16Handler(IMemory memory, BiosDataArea biosDataArea,
         _biosKeyboardBuffer = biosKeyboardBuffer;
         AddAction(0x00, GetKeystroke);
         AddAction(0x01, () => GetKeystrokeStatus(true));
-        AddAction(0x02, () => GetShiftFlags());
+        AddAction(0x02, GetShiftFlags);
+        AddAction(0x1D, () => Unsupported(0x1D));
+    }
+
+    private void Unsupported(int operation) {
+        if (LoggerService.IsEnabled(LogEventLevel.Warning)) {
+            LoggerService.Warning(
+                "{ClassName} INT {Int:X2} {operation}: Unhandled/undocumented keyboard interrupt called, will ignore",
+                nameof(KeyboardInt16Handler), VectorNumber, operation);
+        }
+
+        //If games that use those unsupported interrupts misbehave or crash, check if this state is the proper way to
+        //fix it as I couldn't find documentation about it
+        State.CarryFlag = true;
```

Comment:
Unless documentation can be found on this call, I think it's better to remove those two lines.

Besides, I do not see DOSBox do this ?

I would remove those two lines that sets registers, to avoid causing a crash or potentially weird behavior.

Reply:
I removed setting the flag/register. Will check if another game uses this interrupt and how it behaves.
Thats why I left the log on WARNING as the current treatment may not be correct.

Reply:
Nothing at all in the RBIL or BIOS source codes about this call. Weird.